src/status_im/chat/models/loading.cljs

```diff
@@ -101,6 +101,12 @@
                                     :on-success #(re-frame/dispatch
                                                   [::mark-all-read-in-community-successful %])}]}))
 
+(defn mark-albums [messages]
+  (let [new-messages (atom [])]
+  (doseq [message messages]
```

Comment:
map should be used instead

Reply:
again could this be done on status-go side ?

Reply:
https://github.com/status-im/status-mobile/pull/14635#discussion_r1057427677